packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -1041,6 +1042,128 @@ curl -v -X POST \
 }
 ```
 
+#### Add Smart Card Identity Provider
+
+
+Adds a new Smart Card `X509` type IdP to your organization
+
+##### Request Example
+
+> **Notes:** You must first add the IdP's server certificate to the IdP key store before you can add a Smart Card `X509` IdP with a `kid` credential reference. You need the upload the whole trust chain as single key using [key store apis](#add-x-509-certificate-public-key).
+Depending on the information stored in the smart card select proper template "idpuser.subjectAltNameEmail" or "idpuser.subjectAltNameUpn".
```

Comment:
What does this actually mean? Neither of these templates is explained anywhere, as far as I can tell.

Reply:
The templates are coming from here: 
https://developer.okta.com/docs/reference/okta-expression-language/#idp-user-profile